### PR TITLE
[UI] Drop the solution page and related test

### LINKED ISSUE #3168: Remove the Environment page from the MetalK8s UI
**Component**:

'ui', 'environments'

**Why this is needed**:

All the environment and solution functionalities are not mature enough currently and may need deep brainstorming to better define the set of functionalities we want to provide as part of MetalK8s. As such, even if we keep the solutions tooling available in CLI (for importing images and CRDs as well as deploying the solution operator in a specific namespace), we prefer to remove it from the MetalK8s UI to avoid someone doing wrong actions easily.

**What should be done**:

Remove the Environment page from the MetalK8s UI, its code, and the associated cypress tests.

**Implementation proposal** (strongly recommended):

**Test plan**:

### PR DESCRIPTION
**Pull request by ChengYanJin**
**Component**: ui, solutions

**Context**: 
Since the solution UI is not really useful, we decided to completely drop this page as well as the codebase. 

**Summary**:
Removed:
- Solution page
- Solution-related test, includes e2e solution in CI as well. 
- Sidebar solution entry
- Remove all the related translations

**Acceptance criteria**: 
No regression 

closes: #3168

### CODE REVIEW

**Review by JBWatenbergScality — DISMISSED** [review]

**Comment by JBWatenbergScality** [comment]
Maybe we should mention this change in the changelog as well.

**Comment by ChengYanJin** [comment]
> Maybe we should mention this change in the changelog as well.

Yes. Indeed, I will add a changelog entry.

**Review by JBWatenbergScality — DISMISSED** [review]

File: ui/src/translations/fr.json [hunk 1]
```diff
@@ -136,25 +125,7 @@
   "value": "Valeur",
   "prometheus_not_available": "Prometheus n'est pas disponible.",
   "please_refer_to": "Veuillez vous référer à",
-  "import_solution_hint": "Vous devez importer une solution pour préparer votre environnement",
   "redirecting": "Redirection…",
-  "documentation": "Documentation",
```
Comment (JBWatenbergScality) [inline ui/src/translations/fr.json:141]: Navbar tests are failling because I think this entry should not have been removed

Comment (ChengYanJin) [reply ui/src/translations/fr.json:141]: I just realized it as well! Re-adding. Thanks for checking!

**Review by JBWatenbergScality — DISMISSED** [review]

File: eve/main.yml [hunk 2]
```diff
@@ -2537,12 +2537,6 @@ stages:
...
           env:
             SOLUTION_ARCHIVE: /var/tmp/example-solution-next.iso
-      - ShellCommand:
-          <<: *bastion_ui_tests
-          name: Run Solutions UI tests
-          env:
-            <<: *_env_bastion_ui_tests
-            TEST_FILTER: "e2esolutions"
       - ShellCommand: *collect_cypress_result_ssh
       - Upload: *upload_cypress_artifacts
```
Comment (TeddyAndrieux) [inline eve/main.yml]: I think you want to remove those two as well since we do no longer run any cypress tests

Comment (ChengYanJin) [reply eve/main.yml]: Thanks @TeddyAndrieux! 
Updated in https://github.com/scality/metalk8s/pull/3167/commits/84506135f63f9b7ff12e07de31a8a4d6c41bc856

File: eve/main.yml [hunk 3]
```diff
@@ -2537,14 +2537,6 @@ stages:
           name: Import Solution next example-solution archive
           env:
             SOLUTION_ARCHIVE: /var/tmp/example-solution-next.iso
```
Comment (TeddyAndrieux) [inline eve/main.yml]: Oh btw you also want to remove the 2 imports right before that are only used for this cypress tests (sorry I forgot :smile: )

Comment (ChengYanJin) [reply eve/main.yml]: Hi @teddy!

Thanks for the double-check!
https://github.com/scality/metalk8s/pull/3167/commits/55b1e01da3dc85430493501c5ba648006b3d993b

File: eve/main.yml [hunk 4]
```diff
@@ -2522,29 +2522,6 @@ stages:
             <<: *_env_bastion_tests
             PYTEST_FILTERS: "solution"
       - SetPropertyFromCommand: *set_bootstrap_cp_ip_ssh
```
Comment (TeddyAndrieux) [inline eve/main.yml]: Sorry @ChengYanJin
This one also
(It's the last one :smile: )

Comment (ChengYanJin) [reply eve/main.yml]: Thanks, @TeddyAndrieux !!!
Would be great if you can check if it looks good to you and give your approval 😃 
https://github.com/scality/metalk8s/pull/3167/commits/640cee8b92d69a9d7efbdffb7419bf45b243c473

**Review by TeddyAndrieux — APPROVED** [review]
CI point of view, LGTM now :smile:

**Review by JBWatenbergScality — APPROVED** [review]

**Pull request merged**